eth/block-validation/api.go

```diff
@@ -344,3 +346,89 @@ func (b *BlockAssemblerRequest) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+func (api *BlockValidationAPI) BlockAssembler(params *BlockAssemblerRequest) (*capella.ExecutionPayload, error) {
+	log.Info("BlockAssembler", "tobTxs", len(params.TobTxs.Transactions), "robPayload", params.RobPayload)
+	transactionBytes := make([][]byte, len(params.TobTxs.Transactions))
+	for i, txHexBytes := range params.TobTxs.Transactions {
```

Comment:
why is this copy needed ?

Reply:
type conversions from `[]bellatrix.Transaction` to `[][]byte`. Lot of opaque types here dude. Need to check if there is  cleaner way to do it.